R/AOO_functions.R

```diff
@@ -199,12 +198,25 @@ getMinAOO <- function(ecosystem.data, grid.size,
                  min.percent.rule = min.percent.rule,
```

Comment:
`objective_function` takes a vector 
run `print(o)` before the function exits to see, $par is the argmin